src/main/java/org/folio/circulation/support/MultipleRecordFetcher.java

```diff
@@ -58,7 +58,7 @@ public MultipleRecordFetcher(
   private CompletableFuture<Result<MultipleRecords<T>>> findByBatchQueriesAndQuery(
       List<Result<CqlQuery>> queries, Result<CqlQuery> andQuery) {
     return findByBatchQueries(queries.stream().map(query ->
-      andQuery.combine(query, CqlQuery::and))
+      query.combine(andQuery, CqlQuery::and))
```

Comment:
Why has the order of these been changed?

Reply:
Because the query wasn't built correctly. The sort was in the middle of the query. Also this method wasn't called by anyone so these changes can't be influence to other parts of the code.

Reply:
Hmm, that seems like a bug in the `CQLQuery` class. It would be good if that could be resolved, as the sort should always come at the end, irrespective of the other clauses.

I understand if that is out of scope for this issue. It would be useful to get a demonstration of this in a separate branch if possible.